pkg/install/generate.go

```diff
@@ -154,6 +154,14 @@ func InstallerPodSpec(
 			Name:  "PULLSECRET_SECRET_NAME",
 			Value: constants.GetMergedPullSecretName(cd),
 		},
+		{
+			Name:  "SSL_CERT_DIR",
+			Value: "/etc/ssl/certs/:/output/hive-trusted-cabundle/",
```

Comment:
Per offline discussion, this is what makes it okay to change the cert path and remove `update-ca-trust` from `InstallCerts` :+1: